gatsby/content/blog/2020/08/2020-08-27-synapse-1.19.1-released.mdx

```diff
@@ -0,0 +1,35 @@
+---
+date: '2020-08-27'
+title: Synapse 1.19.1 released
+categories:
+  - Releases
+author: Brendan Abolivier
+---
+
+Synappse 1.19.1 is a bug fix release to fix two bugs introduced in 1.19.0.
+
+Those two bugs were related to the new rate-limiter on room joins, which was introduced in 1.19.0.
```

Comment:
```suggestion
Those two bugs are related to the new rate-limiter on room joins, which was introduced in 1.19.0.
```

Reply:
I'd like to keep consistency across the post, if I'm talking about the bugs in the past later then it makes more sense that this sentence is also in the past